cli/dev/dependency.go

```diff
@@ -112,19 +112,19 @@ func DependencyFetch(homeDirectory string, project string, specRef string, url s
 	// Check input parameters for correctness.
 	dir, err := verifyFetchInput(homeDirectory, project, specRef, url, org, version, arch, userCreds)
 	if err != nil {
-		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, "'dependency %v' %v", DEPENDENCY_FETCH_COMMAND, err)
+		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("'dependency %v' %v", DEPENDENCY_FETCH_COMMAND, err))
```

Comment:
No need to i18ning this string because `dependency` is the subcommand name. Same for all the changes in this file.